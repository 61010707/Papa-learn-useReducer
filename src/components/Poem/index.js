import React from "react";

import styled from "styled-components";

const Main = styled.h2`
  color: whitesmoke;
`;
export default props => {
  return (
    <Main>
      Rose are {props.color}, violet are {props.color}
    </Main>
  );
};
