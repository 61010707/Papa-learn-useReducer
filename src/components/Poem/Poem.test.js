import React from "react";
import { shallow } from "enzyme";

import Poem from "./index.js";

describe("Poem", () => {
  it("should render ", () => {
    const component = shallow(<Poem />);
    expect(component).toMatchSnapshot();
  });
});
