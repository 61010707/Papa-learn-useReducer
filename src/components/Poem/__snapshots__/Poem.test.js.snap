// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Poem should render  1`] = `
<styled.h2>
  Rose are 
  , violet are 
</styled.h2>
`;
